platform/ios/MGLMapView.mm

```diff
@@ -285,9 +285,6 @@ - (void)commonInit
 
     _annotationImages = [NSMutableDictionary dictionary];
 
-    std::string defaultSymbolName([MGLDefaultStyleMarkerSymbolName UTF8String]);
```

Comment:
If `MGLDefaultStyleMarkerSymbolName` is no longer needed, it doesn’t need to be declared up top anymore.